hack/ci/e2e.sh

```diff
@@ -202,7 +223,7 @@ EOF
     fi
 
     # add ginkgo args
-    KUBETEST_ARGS="${KUBETEST_ARGS} --test_args=\"--ginkgo.focus=${FOCUS} --ginkgo.skip=${SKIP} --report-dir=${ARTIFACTS} --disable-log-dump=true --num-nodes=${NUM_NODES}\""
+    KUBETEST_ARGS="${KUBETEST_ARGS} --test_args=\"--ginkgo.focus=${FOCUS} --ginkgo.skip=${SKIP} --container-runtime=containerd --report-dir=${ARTIFACTS} --disable-log-dump=true --num-nodes=${NUM_NODES}\""
```

Comment:
this coupling is :(

Reply:
this was because of https://github.com/kubernetes/kubernetes/issues/80098 , but reading the issue seems that is not the problem

Reply:
ah yes, the problem is the test-handler, what happens if we remove this flag?

Reply:
it doesn´t affect the other tests